lib/trace.js

```diff
@@ -461,14 +461,16 @@ Trace.prototype.getLoadRecord = function(canonical, excludeURLs, parentStack) {
             if (typeof sourceMap == 'string')
               sourceMap = load.metadata.sourceMap = JSON.parse(sourceMap);
 
-            var originalName = load.name.split('!')[0];
-            
-            // force set the filename of the original file
-            sourceMap.file = originalName + '!transpiled';
-
-            // force set the sources list if only one source
-            if (!sourceMap.sources || sourceMap.sources.length <= 1)
-              sourceMap.sources = [originalName];
+            if (load.name.indexOf('!') != -1) {
```

Comment:
> Are you only using input source maps with plugins loaded with ! syntax?

Not quite. The input source maps work regardless of this check, but this code changes the file names, which makes source maps useless, so now it only runs for plugins. I'm not sure if this is correct and what was the original intention or how my change will affect this.
I have a couple of examples in this PR:
* `register.js` has source map indicating that `babel.js` is its original source, but this code changes it to `register.js!transpiled → register.js`
* similarly `example.js → example` becomes `example.js!transpiled → example.js`